PointOfDispensingApp/config.js

```diff
@@ -0,0 +1,10 @@
+//file name is 'config.js'
+
+export default {
+    vaccinationDecisionState:
+    {
+        UNDETERMINED: 0,
+        VACCINATION_PROCEED_YES: 1,
+        VACCINATION_PROCEED_NO: 2   
```

Comment:
Is there a reason we use underscores and not PascalCase? These names are also just feeling wrong... this data can be a bool as a state variable we reference on the page.

I'd understand more if we're creating status flags to indicate where they are in the workflow, but at that point these don't make sense there. What're your thoughts on this?

Reply:
I can change it - do we have a convention for this?  I thought it was relatively standard to have all caps and underscores for defines and enum values.

The reason why I didn't go with a bool is because there are 3 states (which is maybe becoming more clear from my above comments):
0: decision about whether or not vaccination will occur has not been decided
1: decision has been made to proceed with vaccination
2: decision has been made to cancel the vaccination